src/SmartComponents/BaselinesTable/redux/actions.js

```diff
@@ -1,10 +1,10 @@
 import types from './types';
 import api from '../../../api';
 
-function fetchBaselines(params = {}) {
+function fetchBaselines(params = {}, isModal) {
```

Comment:
I think the idea of `isModal` is bleeding concepts into the layers that shouldn't care if it's a modal or not.